### PR TITLE
feat: information page of specific club & autoclean HTML `<script>` `<style>` elements

### PR DESCRIPTION
**Pull request by at-wr**
This PR added a new feature which includes a new page of displaying some of the details of a specific club after clicking the button of a specific club in `/cas/clubs`.
Though it has introduced a new page, however the style needs to be improved and more information is required in the future PRs.

Related Issue: https://github.com/Computerization/Enspire/issues/485

### CODE REVIEW

**Comment by at-wr** [comment]
Please don't merge this PR before I fixed the Linter issue

**Comment by at-wr** [comment]
> Please don't merge this PR before I fixed the Linter issue

Linter issue fixed

File: components/custom/club-card.vue
Comment (qwerzl) [inline components/custom/club-card.vue]: 1. Use `NuxtLink` for better code style. 
2. Add a `w-full` class to expand the width.

File: pages/clubs/[id].vue
Comment (qwerzl) [inline pages/clubs/[id].vue]: Add a `space-x-2` class to add space between the cards.

File: pages/clubs/[id].vue
Comment (qwerzl) [inline pages/clubs/[id].vue]: Try to be responsive. Only set w-3/4 on larger screens.

File: pages/clubs/[id].vue
Comment (qwerzl) [inline pages/clubs/[id].vue]: Same as above. Try hiding it on smaller screens.

File: pages/clubs/[id].vue
Comment (qwerzl) [inline pages/clubs/[id].vue]: 1. Only show `C_DescriptionC`. Add `C_DescriptionE` until we have added support for i18n.

2. Show "暂无简介" if `C_DescriptionC` is empty. 

3. A question for you to decide: Some clubs like the student union add HTML codes to the club description. We should clean the up by regex. Would you prefer cleaning it in the python script (of crawling the clubs) or cleaning it in the frontend?

**Review by qwerzl — CHANGES_REQUESTED** [review]

File: pages/clubs/[id].vue
Comment (at-wr) [reply pages/clubs/[id].vue]: My answer to the 3rd question:

I would prefer cleaning it in the Python script. Since certain JavaScript scripts / CSS styles will only be available on TSIMS, processing the text on the frontend will not only increase the workload of clients' browser, but it will also create an amount of unused network traffic.

Comment (at-wr) [reply pages/clubs/[id].vue]: Well, I've implemented such features with cleanHTML() function, so it can be fully accomplished via frontend rn

**Comment by at-wr** [comment]
@qwerzl The requests have been fullfilled

**Comment by at-wr** [comment]
I think it looks great rn

File: components/custom/cleanHTML.d.ts
Comment (qwerzl) [inline components/custom/cleanHTML.d.ts]: Move it to `utils` directory

File: pages/clubs/[id].vue
Comment (qwerzl) [inline pages/clubs/[id].vue]: Use `v-if` for that. `v-html` might introduce security issues afaik

File: pages/clubs/[id].vue
Comment (qwerzl) [inline pages/clubs/[id].vue]: Move it above the previous card so that the "社团属性“ card stays above it on small screens.

File: pages/clubs/[id].vue
Comment (qwerzl) [inline pages/clubs/[id].vue]: Make ”社团类型“ bold

File: pages/clubs/[id].vue
Comment (qwerzl) [inline pages/clubs/[id].vue]: Make ”社团人数“ bold

File: pages/clubs/[id].vue
Comment (qwerzl) [inline pages/clubs/[id].vue]: Make "指导老师” bold

**Review by qwerzl — CHANGES_REQUESTED** [review]
Awesome. These are the final changes requested before I merge it.

**Review by qwerzl — APPROVED** [review]

**Pull request merged**
